webservices/src/main/java/com/javaschool/webservices/service/impl/PackageServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.javaschool.webservices.service.impl;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.javaschool.webservices.model.PackageType;
+import com.javaschool.webservices.service.PackageService;
+import com.javaschool.webservices.service.ShippingService;
+
+@Service
+public class PackageServiceImpl implements PackageService {
+	
+	private final ShippingService shippingService;
+	
+	public PackageServiceImpl(ShippingService shippingProducerService) {
+		this.shippingService = shippingProducerService;
+	}
+
+	@Override
+	public List<String> getSizes() {
+		List<String> sizesList = new LinkedList<>();
+		
+		sizesList.add("SMALL");
+		sizesList.add("MEDIUM");		
+		sizesList.add("LARGE");
+		sizesList.add("EXTRA LARGE");
+		
+		return sizesList;
+	}
+
+	@Override
+	public List<String> getTypes() {
+		List<PackageType> packageTypes = shippingService.getPackageTypes();
+		
+		List<String> packageTypesNames = new LinkedList<String>(); 
```

Comment:
instead of create a new list you can use:
```
packageTypes.stream()
	.map(packageType->packageType.getDescription())
	.collect(Collectors.toList());
```

Reply:
use of stream instead of create a new instance of a list.